src/myCounter.js

```diff
@@ -11,11 +11,11 @@
 */
 
 const myCounter = () => {
-  var myArray = [];
-  for (var counter = 0; counter <= 3; counter += 1) {
+  let myArray = [];
```

Comment:
Sugiro utilizar `const` ao invés de `let` nessa linha, já que o array não é alterado.

```suggestion
  const myArray = [];
```